src/include/pbs_ifl.h

```diff
@@ -466,7 +466,7 @@ enum mgr_obj {
 #define PBS_MAXPWLEN		256		/* max password length */
 #define PBS_MAXGRPN		256		/* max group name length */
 #define PBS_MAXQUEUENAME	15		/* max queue name length */
-#define PBS_MAXJOBNAME  	230		/* max job name length */
+#define PBS_MAXJOBNAME  	MAXPATHLEN + 1	/* max job name length */
```

Comment:
This is not right... PBS_MAXJOBNAME is what we can pass in `-N` in qsub/pbs_rsub, not complete path to script. see `decode_jobname()` action func... Please revert this.